examples/step-98/doc/intro.dox

```diff
@@ -0,0 +1,60 @@
+This tutorial program, step-98, demonstrates the implementation and application of geometric multigrid (GMG) methods
+combined with patch smoothers with localized residual computations. It builds upon the concepts introduced in step-37.
+
+The work is based on the algorithms described in the following paper:
+<br>
+<i>
+Smoothers with localized residual computations for geometric multigrid methods
+<br>
+Michał Wichrowski, Peter Munch, Martin Kronbichler, Guido Kanschat
+<br>
+https://arxiv.org/abs/2407.02100
+</i>
+
+<a name="Motivation"></a>
+<h1>Motivation</h1>
+
+Geometric multigrid methods are highly efficient solvers for the large linear systems arising from the discretization of
+partial differential equations. A key component of GMG is the smoother, typically an iterative method like Jacobi
+or Chebyshev iteration, which damps high-frequency error components on each level of the multigrid hierarchy.
+However, these methods  rely on global operations: such as computing residuals or multipying with diagonal.
+These operations can lead to poor CPU cache utilization due to the non-local memory access patterns, especially for
+large-scale problems. Furthermore, the number of iterations typically grows significantly with the polynomial degree ($p$) of the finite
+element space.
+
+
+To address this challenge, this program explores a specific type of localized smoother: a **patch smoother**. The core
+idea is to avoid global residual computations within the smoother step. Instead, the smoother operates on small,
+overlapping vertex-patches of cells (consisting of $2^d$ cells). For each patch, the residual is computed using only the
+solution values within that patch, and an approximate inverse of the operator restricted to that patch is applied to the
+local residual to compute a correction. This localization significantly reduces the cache overhead during the
+smoothing process compared to standard smoothers relying on global matrix-vector products. For structured
+patches (like those arising from uniform refinement), the local patch operator often exhibits a tensor product structure,
+allowing for highly efficient computation and inversion using specialized techniques.
+
```

Comment:
I added reference to step-59, I am not putting it in builds-on: understaning DG is definately not required for this tutourial.